### DISCUSSION TITLE
Display more info on (own) POI (in the map)

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#512] [Q&A] [answered]
Since POI Icons are just displayed in the map (or do I have missed something?)
(see example picture)
![image](https://github.com/devemux86/cruiser/assets/57968526/9d3f1b72-91aa-4916-80df-7bb9dfa46466)

...and only show name by clicking on ,
![image](https://github.com/devemux86/cruiser/assets/57968526/2aca491c-e9ef-4149-9532-72cbb4f94852)

 can it be possible to see at least the name (permanently by setting activation) in the map next to the POI? - It can be then even smaller. 
Instead of the picture above - rather like the following one: (example construction: text size, type display)
![image](https://github.com/devemux86/cruiser/assets/57968526/7ddab4e7-4974-448b-b9fd-567636947a90)

A "click on icon" function should still be possible and connected to this:
Is it then even possible to show more info with a click, like not only name but:
`<ele>` (which is elevation - when available)    
`<cmt>` (wich ich comment - when available)

which could then look like (example construction)
![image](https://github.com/devemux86/cruiser/assets/57968526/c237ee2f-6091-4db3-9e21-9fbfb2105413)

If - especially the last example can be possible, it might be possible as well - which additional info should be then displayed once the are available in the gpx data. i.e. not "elevation" but "comment" etc.

**Comment by devemux86** [comment, marked as answer]
@DIrgi thanks for the suggestion.

(currently the name appears with a tap on the favorite, as with all overlays)

It could be through an option in the favorite groups, i.e. which group has a visible label.

For additional fields in favorites, we can discuss it here:
- https://github.com/devemux86/cruiser/discussions/346

**Reply by bios4** [reply]
Please show these descriptions only OPTIONALLY with a setting!

I definitely don't want to have the map plastered with POI names/descriptions... 😜

**Reply by DIrgi** [reply]
If you read closely nothing other has been said in the proposal 😉

**Reply by bios4** [reply]
Yes - yet I again wanted to emphasize this... 😉
